helm/oauth2-proxy/Chart.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: redis
   repository: https://charts.bitnami.com/bitnami
-  version: 18.3.2
-digest: sha256:c4d7208b098ca3c44f960f02fb0e505ac6493cf00c6539ee7bf0a603371d9959
-generated: "2023-11-13T12:24:54.470708+01:00"
+  version: 18.3.3
```

Comment:
Please put the Redis upgrade in a separate PR.

Reply:
We already have 1 PR for that: #179

Reply:
fixed!